openedx/core/lib/block_cache/block_structure.py

```diff
@@ -13,6 +15,16 @@
 TRANSFORMER_VERSION_KEY = '_version'
 
 
+def zpickle(data):
+    """Given any data structure, returns a zlib compressed pickled serialization."""
+    return zlib.compress(pickle.dumps(data, pickle.HIGHEST_PROTOCOL))
+
+
+def zunpickle(zdata):
+    """Given a zlib compressed pickled serialization, returns the deserialized data."""
+    return pickle.loads(zlib.decompress(zdata))
```

Comment:
hmm.. we should think about putting this in a more general location.
Perhaps cache_utils.py in  `edx-platform/openedx/core/lib/`?